java/vector/src/test/java/org/apache/arrow/vector/TestSplitAndTransfer.java

```diff
@@ -209,6 +225,39 @@ public void test() throws Exception {
     }
   }
 
+  @Test /* ViewVarCharVector */
+  public void testView() throws Exception {
```

Comment:
Instead of the comment, just rename the test?

Reply:
Ah yes, I should have got rid of that comment, already renamed to views.